Jenkinsfile

```diff
@@ -52,15 +52,19 @@ pipeline {
             deleteDir()
             gitCheckout(basedir: "${BASE_DIR}", githubNotifyFirstTimeContributor: true)
             stash allowEmpty: true, name: 'source', useDefaultExcludes: false
-            // Look for changes related to the benchmark, if so then set the env variable.
             script {
               dir("${BASE_DIR}"){
+
+                // Look for changes related to the benchmark, if so then set the env variable.
                 def regexps =[
                   '^packages/.*/test/benchmarks/.*',
                   '^scripts/benchmarks.js',
                   '^packages/rum-core/karma.bench.conf.js'
                 ]
                 env.BENCHMARK_UPDATED = isGitRegionMatch(patterns: regexps)
+
+                // Skip all the stages except docs for PR's with asciidoc changes only
+                env.ONLY_DOCS = isGitRegionMatch(patterns: [ '.*\\.asciidoc' ], comparator: 'regexp', shouldMatchAll: true)
```

Comment:
`shouldMatchAll: true` will help to filter those changes that contain code changes.

Reply:
do we need the `comparator` here?  Asking because the benchmarks regex didn't have it.

Reply:
Good question, the `regexps` variable name is a bit misleading in the benchmark, the benchmark is not a regex but a glob. In other words,  `isGitRegionMatch` got the default comparator as glob rather than regexp. I'll change the variable name to ensure we don't get confused in the future.

Reply:
Thanks Victor for the details. Sounds good.